### PR TITLE
LPAR activate C sample

### PR DESCRIPTION
**Pull request by gorelikg**

### CODE REVIEW

File: Example-LPARActivate-C/cpp/hwirstc1.cpp [hunk 1]
```diff
@@ -0,0 +1,1163 @@
...
+  return jobRunning;
+}
+
+/*
+ * Method: printConstTextStr
+ *
+ * Print out the contents of the constant string if the string
+ * length is greater than 0
+ */
+void printConstTextStr(int len, const char *text, char *description)
```
Comment (blairwyman) [inline Example-LPARActivate-C/cpp/hwirstc1.cpp:924]: Calls to this function could be safely replaced with `printf("* >%s: %.*s", description, len, text);`

File: Example-LPARActivate-C/cpp/hwirstc1.cpp [hunk 2]
```diff
@@ -0,0 +1,1163 @@
...
+* Unless required by applicable law or agreed to in writing,          *
+* software distributed under the License is distributed on an         *
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,        *
+* either express or implied.  See the License for the specific        *
+* language governing permissions and limitations under the License.   *
+*                                                                     *
+* End of Copyright and License                                        *
+***********************************************************************
+*                                                                     *
+*    MODULE NAME= HWIRTC1                                             *
```
Comment (bnoordyke) [inline Example-LPARActivate-C/cpp/hwirstc1.cpp:21]: The Module Name needs an 'S' in it.

**Review by bnoordyke — CHANGES_REQUESTED** [review]
Update the Module Name in the prologue in hwirstc1.cpp.

**Review by bnoordyke — COMMENTED** [review]
Looks good to me.

**Pull request merged**